components/Person/PersonDetail.js

```diff
@@ -79,125 +53,171 @@ const PersonUl = styled.ul`
 `
 
 const PersonDetail = ({ person }, ...props) => (
-  <ProfileGrid>
-    <GridContainer> {/* Left Sidebar */}
-      <Head title={person.nickname} />
-      <ProfileImage src={person.imgUrl} alt={person.nickname} />
-      {person.orgMembership &&
-        <DetailItem>
-          <H4>
-            <FormattedMessage
-              id='PersonDetail.subheading.membership'
-              defaultMessage='Member of'
-              description='Header for list of orgs I belong to'
-            />
-          </H4>
-          <MemberUl members={person.orgMembership} />
-        </DetailItem>}
-      {person.orgFollowership &&
-        <DetailItem>
-          <H4>
-            <FormattedMessage
-              id='PersonDetail.subheading.following'
-              defaultMessage='Following'
-              description='Header for list of orgs I follow'
-            />
-          </H4>
-          <MemberUl members={person.orgFollowership} />
-        </DetailItem>}
-      <DetailItem>
-        <H4>
-          <FormattedMessage
-            id='PersonDetail.subheading.achievements'
-            defaultMessage='Recognition'
-            description='Header for list of badges I have obtained'
-          />
-        </H4>
-        <PersonBadgeSection person={person} />
-      </DetailItem>
-    </GridContainer>
-    <GridContainer> {/* Main Workspace */}
-      <InfoSection>
-        <H1>{person.name}</H1>
-        <Divider />
-        <Markdown children={person.about || ''} />
-      </InfoSection>
+  <div>
+    <Head title={person.nickname} />
+    <VBanner>
+      <VBannerImg src={person.imgUrl} alt={person.nickname} />
+      <ProfileBannerTitle>
+        <h1>{person.name}</h1>
+
+        <p>{person.job && `${person.job}`} {person.placeOfWork && `- ${person.placeOfWork}`}</p>
+
+      </ProfileBannerTitle>
+    </VBanner>
+    <Divider />
+    <ActivityContainer>
+      <h2>About</h2>
+      <div>
+        {person.about &&
+          <div>
```

Comment:
please use <> rather than <div> is the wrapper is simply to group the components.